docs/API.md

```diff
@@ -257,38 +216,6 @@ const myInput = await $('input')
 await expect(myInput).toHaveAttr('class', 'form-control')
 ```
 
-### toHaveAttributeContaining
-
-Checks if an element has a certain attribute that contains a value.
-
-__Deprecated:__ please use `expect.stringContaining()` instead!
-
-##### Usage
-
-```js
-const myInput = await $('input')
-// 👎 deprecated
-await expect(myInput).toHaveAttributeContaining('class', 'form')
-// 👍 supported starting with v9
-await expect(myInput).toHaveAttribute('class', expect.stringContaining('form'))
```

Comment:
Could we add this as example to the matchers so people see how they can assert against the full or partial text.